modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinConnectionSelfTest.java

```diff
@@ -1720,6 +1720,19 @@ public void testRollbackSavePoint() throws Exception {
         }
     }
 
+    /**
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testDisabledFeatures() throws Exception {
+        assertInvalid(url + "?disabledFeatures=unknownFeature",
+            "Unknown feature: unknownFeature");
+
+        try (Connection conn = DriverManager.getConnection(url + "?disabledFeatures=reserved")) {
```

Comment:
Empty code block. Do you miss any asserts? If not please add comment:
// No-op.

Reply:
No, it was done intentionally. This part verifies that with allowed parameters you could connects to the grid. I have added the comment.